src/utils/tokens.ts

```diff
@@ -46,3 +47,17 @@ export const getTokenDecimalsWithFallback = (
   }
   return DEFAULT_TOKEN_DECIMALS;
 };
+
+export const getFormattedTokenValue = (
+  value: BigNumberish,
+  decimals: any,
+): string => {
+  const tokenDecimals = getTokenDecimalsWithFallback(decimals);
+  const safeDecimals = bigNumberify(10)
+    .pow(getTokenDecimalsWithFallback(tokenDecimals))
```

Comment:
Im lost a bit, why you fire getTokenDecimalsWithFallback twice?

Reply:
My bad, this slipped my mind and I forgot to remove it